src/script/engine.zig

```diff
@@ -1025,3 +1026,58 @@ test "Script execution - OP_SHA256" {
     Sha256.hash(&[_]u8{1}, &expected_hash, .{});
     try std.testing.expectEqualSlices(u8, expected_hash[0..], hash_bytes);
 }
+
+test "ScriptBuilder Smoke test" {
+    var sb = try ScriptBuilder.new(std.testing.allocator, null);
+    defer sb.deinit();
+
+    var engine = try (try (try (try sb.addInt(1)).addInt(2)).addOpcode(Opcode.OP_ADD)).build();
+
+    try engine.execute();
+
+    defer engine.deinit();
+
+    try std.testing.expectEqual(1, engine.stack.len());
+}
+
+//METHOD 1
+test "ScriptBuilder OP_SWAP METHOD 1" {
```

Comment:
you can also remove those two following tests.

Unittest are for testing ONE thing. Here we wan to know if the ScriptBuilder works. So we give it an input, and then look at it's output. 
In those tests you don't look at the output at all and instead feed it to another part of the program that may contain it's own bug. If something goes wrong it can either be in the script builder or the OP_SWAP method